tools/ichaindis.py

```diff
@@ -19,11 +20,53 @@
     'ICHAIN_VEC3S',
 ]
 
+Z64_ACTOR_PATH = "../include/z64actor.h"
+
+def get_rom_address(offset):
+    # Run the sym_info script and parse out the ROM address
+    stream = os.popen('cd .. && python3 sym_info.py ' + offset)
```

Comment:
I would prefer if our python scripts called each others' functions directly rather than spawning a new python process like this

Reply:
Weird.  I had already made this change but put it in the wrong branch.  I've applied the changes now.  I couldn't convince Python to include a script from the parent folder so I refactored the sym_info.py script to move the ROM lookup function into its own file in the tools folder.